src/bindgen/library.rs

```diff
@@ -474,25 +481,49 @@ impl Library {
                     }
                 }
             }
+
+            for generic_param in enum_.generic_params.0.iter_mut() {
```

Comment:
make it a function, and you haven't covered all of `generic_params'. Do I really have to go and tell you each occurrence?